recipients_app/docu/app_testing_guides/manual_test_plans.md

```
@@ -68,6 +68,22 @@ corresponding
 - Can you fill out the survey?
 - If you filled out a survey, then the survey is shown as “Completed”.
 
+## Check if database is not in the Android backup files
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)